app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{self.email = email.downcase}
+  validates :name,  presence: true, length: {maximum: 50}
```

Comment:
Các giá trị `5`, `255`, `60` thì khai báo vào file setting nào
Tìm hiểu `gem config in rails` để làm được điều trên nhé